v03_pipeline/lib/reference_datasets/splice_ai.py

```diff
@@ -49,5 +49,6 @@ def get_ht(
         ),
     ).drop('delta_scores')
     return ht.group_by(*ht.key).aggregate(
+        delta_score=hl.agg.max(ht.delta_scores),
```

Comment:
so the aggregation is now happening twice in an inconsistent way, which I do not think is correct. Essentially, its parsing the VCF into a list of scores per site, then aggregating those scores, then at the end its doing a group by here and re-aggregating. I think this is unliklely to be producing the correct result. I think we need to do a quick investigation into places where we do have multiple sites after the VCF is initially read in an see if there really are different `ht.info.SpliceAI` values for the exact same site multiple times in the vcf. If not and theres just weird duplication a call to `distinct` would be better than a `group_by`, and if there are then we need to be explicitly deciding which of the duplicate entries to use

Also this shouldn't work because `delta_scores` is dropped on the line above this

Reply:
I'm looking into this more thoroughly.  This was a solution but not a good one.

Reply:
Looks like the duplicates (that I could find) are related to gene/transcripts.  I think the easiest path forwards is to filter to duplicate rows and remove those with a `info.SpliceAI[1]` that contains a `.`.  There are instances of transcript scores that do not have corresponding gene rows.  
````
+---------------+------------+------+-----------+----------+------------------------------------------------+
| locus         | alleles    | rsid |      qual | filters  | info.SpliceAI                                  |
+---------------+------------+------+-----------+----------+------------------------------------------------+
| locus<GRCh37> | array<str> | str  |   float64 | set<str> | array<str>                                     |
+---------------+------------+------+-----------+----------+------------------------------------------------+
| 1:861264      | ["C","A"]  | NA   | -1.00e+01 | NA       | ["A|AL645608.1|0.00|0.00|0.00|0.00|2|27|12|1"] |
| 1:861264      | ["C","A"]  | NA   | -1.00e+01 | NA       | ["A|SAMD11|0.02|0.01|0.00|0.00|14|38|14|38"]   |
+---------------+------------+------+-----------+----------+------------------------------------------------+
```